sinonjs/sinon
# Upgrade mocha and mochify

#### Purpose

This is to be rid of the warnings during install about regexp DoS issue.

https://snyk.io/vuln/npm:minimatch:20160620

#### How to verify

1. In `master` branch, `npm install && npm test`
1. Observe the number of tests
1. Check out this branch
1. `npm install && npm test`
1. Observe that all tests pass and that the number of tests are the same as before
